#################################
# PID parameters for pose seeking
#################################

# Maximum value of error integral for all PID controllers
windup_limit: 0.05

parameter_ns: ""

# PID gains
x_proportional_gain: 2.0
y_proportional_gain: 2.0
z_proportional_gain: 2.0
x_integral_gain: 0.00 #0.02
y_integral_gain: 0.00
z_integral_gain: 0.00
x_derivative_gain: 0 #0.0199999999999999999999
y_derivative_gain: 0
z_derivative_gain: 0

angular_proportional_gain: 0.7
angular_integral_gain: 0.007 #0.007
angular_derivative_gain: 0 #0.001

# Pose Tracking tolerances
lin_tol: [0.01, 0.01, 0.01] # [x,y,z]
rot_tol: 0.1

# Target timeout
timeout: 0.1